public/support/mdb_operator_diagnostic_data.sh

```diff
@@ -49,7 +89,7 @@ usage() {
 contains() {
   local e match=$1
   shift
-  for e; do [[ "${e}" == "${match}" ]] && return 0; done
+  for e; do [[ "$e" == "$match" ]] && return 0; done
```

Comment:
is this change necessary? this might upset the shellcheck. or the changed version is better suited for shellcheck? 😆